device/celestica/x86_64-cel_seastone-r0/port-mode.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I feel like this script can be added to sonic-cli like

config select hwsku [hwsku_name]

then it will select the corresponding hardware sku and prompt for rebooting.

then the code can then be used by different platforms.